packages/iran/README.md

```diff
@@ -0,0 +1,49 @@
+# 🇮🇷 Iran
+
+Contains all the states and the Federal District of Iran:
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/Provinces_of_Iran), these are actually provinces.
If you know a better source of information, feel free to share it.